pkg/cli/option/flag.go

```diff
@@ -106,6 +106,8 @@ const (
 	// RoleFlag is the flag name for the role flag.
 	RoleFlag = "role"
 
+	// SharedFlag is the flag name for the shared flag.
+	SharedFlag = "shared"
```

Comment:
This was just an artifact of conflict resolution I think. nbd

```suggestion
	SharedFlag = "shared"

```